### PR TITLE
Ticket14

### PR DESCRIPTION
**Pull request by blizyCodes**
GET api/articles/:article_id 
Tested for:
-article object response
-400 error when there's a bad request for id 
-404 when article id does not exist yet

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -42,3 +42,49 @@ describe("/api/topics", () => {
...
+          .then(({ body: { article } }) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
```
Comment (Angaca) [inline __tests__/app.test.js:63]: This test is not accurate enough, how do you know you are getting exactly the article you expect?

**Review by Angaca — CHANGES_REQUESTED** [review]
Code is good but test needs a bit of adjustment to be more specific

File: __tests__/app.test.js (hunk 1 above)
Comment (blizyCodes) [reply __tests__/app.test.js:63]: I added an extra assertion on that test for it to equal that specific article as per latest commit.

**Review by Angaca — APPROVED** [review]
Great. Good to go!

**Pull request merged**